pulp_smash/tests/rpm/api_v2/test_unassociate.py

```diff
@@ -94,6 +94,32 @@ def test_02_remaining_units(self):
         }
         self.assertEqual(removed_ids & remaining_ids, set())
 
+    def test_03_last_unit_removed(self):
+        """Assert that ``last_unit_removed`` timestamp was not updated.
+
+        Whenever disassociate_units is called ``last_unit_removed`` should not
+        be updated even if that call does not remove units.
```

Comment:
This comment isn't quite right. Whenever `disassociate_units` is called, `last_unit_removed` *should* be updated. The only exception is when no units are removed. This could read:

> If `disassociate_units` is called and no units are removed, `last_unit_removed` should not update.

I'll update this in a follow-up commit if this is left as is, so don't stress out about it.